src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-utils.c

```diff
@@ -277,6 +278,18 @@ long xclmgmt_hot_reset(struct xclmgmt_dev *lro, bool force)
 #else
 		xclmgmt_reset_pci(lro);
 #endif
+		/*
+		 * For U50 built with 2RP flow. PLP Gate is closed after
+		 * pci hot reset. Any access to PLP IPs will hangs the host.
+		 * E.g. read/write BAR2 (DMA BAR). 
```

Comment:
So, a malicious VM can just hang the host by accessing these IPs right after hot reset? I thought this PCIE firewall IP is for preventing this from happening, no?